packages/flutter/lib/src/rendering/mouse_cursor.dart

```diff
@@ -419,7 +418,7 @@ class SystemMouseCursor extends MouseCursor {
 class SystemMouseCursors {
   // This class only contains static members, and should not be instantiated or
   // extended.
-  factory SystemMouseCursors._() => null;
+  factory SystemMouseCursors._() => throw Error();
 
   // The mapping in this class must be kept in sync with the following files in
   // the engine:
```

Comment:
while we're in this file can we change `//// (.+) ////` to `// $1` ? It's messing up the dartdoc generation.

Reply:
Done